src/command/process_update_task.rs

```diff
@@ -344,6 +508,68 @@ impl UpdateProcessor {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, TypedBuilder, Eq, PartialEq, Clone)]
+#[serde(crate = "fang::serde")]
+pub struct ScheduleWeatherTask {
+    user_hour: i8,
+    minutes: i8,
+    offset: i8,
+    username: String,
+    chat_id: i64,
+    default_city_id: i32,
+}
+
+#[typetag::serde]
+#[async_trait]
+impl AsyncRunnable for ScheduleWeatherTask {
+    async fn run(&self, _queueable: &mut dyn AsyncQueueable) -> Result<(), FangError> {
+        // here we should program the weather_info deliver
+        let repo = Repo::repo().await.unwrap();
```

Comment:
I think it's possible to use `Repo::repo().await?`

because errors are automatically converted into FangError

The same comment is for other unwraps in this method

Reply:
[Not unwrap in ScheduleWeatherTask](https://github.com/pxp9/weather_bot_rust/pull/29/commits/7628d104241eb1c62d6014a4030a85b6b00aad7c)